### PR TITLE
Op paygas

### PR DESCRIPTION
**Pull request by SamWilsn**

### CODE REVIEW

File: core/vm/jump_table.go [hunk 1]
```diff
@@ -50,18 +50,33 @@ type operation struct {
...
-	constantinopleInstructionSet   = newConstantinopleInstructionSet()
-	istanbulInstructionSet         = newIstanbulInstructionSet()
+	frontierInstructionSet           = newFrontierInstructionSet()
+	homesteadInstructionSet          = newHomesteadInstructionSet()
+	tangerineWhistleInstructionSet   = newTangerineWhistleInstructionSet()
+	spuriousDragonInstructionSet     = newSpuriousDragonInstructionSet()
+	byzantiumInstructionSet          = newByzantiumInstructionSet()
+	constantinopleInstructionSet     = newConstantinopleInstructionSet()
+	istanbulInstructionSet           = newIstanbulInstructionSet()
+	accountAbstractionInstructionSet = newAccountAbstractionInstructionSet()
```
Comment (lightclient) [inline core/vm/jump_table.go:60]: This is a sad diff.

File: core/vm/opcodes.go [hunk 2]
```diff
@@ -530,6 +537,7 @@ var stringToOp = map[string]OpCode{
 	"LOG2":           LOG2,
 	"LOG3":           LOG3,
 	"LOG4":           LOG4,
+	"PAYGAS":		  PAYGAS,
```
Comment (lightclient) [inline core/vm/opcodes.go]: Looks like there might be an extra tab here.

**Review by lightclient — APPROVED** [review]
Looks good to me.

File: core/vm/opcodes.go (hunk 2 above)
Comment (tecywiz121) [reply core/vm/opcodes.go]: I hate tabs, and I hate horizontal alignment.

Comment (SamWilsn) [reply core/vm/opcodes.go]: I hate tabs, and I hate horizontal alignment.

**Pull request merged**